tmt/steps/provision/connect.py

```diff
@@ -24,6 +24,12 @@ class ConnectGuestData(tmt.steps.provision.GuestSshData):
         metavar='USERNAME',
         help='Username to use for all guest operations.'
         )
+    become: bool = field(
+        default=False,
+        is_flag=True,
+        option=('-b', '--become'),
+        help='Whether to run tests as sudo.'
+        )
```

Comment:
I'd say `connect` inherits from `GuestSshData` to which you already added the field.

Reply:
Thank you @happz , I missed that one. I updated the commit.